jobs/job-scheduler/src/main/java/com/paladincloud/jobscheduler/service/JobScheduler.java

```diff
@@ -92,6 +90,9 @@ public class JobScheduler {
     @Value("${scheduler.role}")
     private String roleName;
     
+    @Value("${plugins.in.v1}")
+    private String pluginUsingV1;
+    
```

Comment:
I believe we are going to use this db config approach for some time... So let's make it better. Let's split values into sections by mean.
Let's say we store ALL plugin schedule related items in `pluginsSchedule` object which will be a map.
so when `ConfigUtil.setConfigProperties();` gets executed it sees them (let's say all of them have prefix `pluginsSchedule.*` and combine them together into a hashmap. And that map is stored as a System property.
This way we don't need to have a list of all plugins we have, we just need to have a `pluginsSchedule` map handy, which will have property `enabledPlugins` - list of all ONLY enabled plugins.
So then instead of running through every of them and check if it's enabled - you will need to go through that list and send SQS for every of them, because list contains ONLY of enabled plugins.

Also we could split that huge config into parts and have separate methods for every parts. List here you need only some general settings and sheduler settings, so there is no need to gather them all, it's waste of resources.

Reply:
What do you this folks? @arunpaladin @plyubich